hardware/raspberrypi/gpio/README.md

```diff
@@ -4,29 +4,29 @@ General Purpose Input/Output pins on the Raspberry Pi
 
 ## Overview
 
-This page expands on the technical features of the GPIO pins available on BCM2835 in general. For usage examples, see [GPIO Usage for A+ and newer](../../../usage/gpio-plus-and-raspi2/README.md) or [GPIO Usage for models A and B](../../../usage/gpio/README.md). When reading this page, reference should be made to the BCM2835 ARM Peripherals [Datasheet](../bcm2835/README.md), section 6.
+This page expands on the technical features of the GPIO pins available on BCM2835 in general. For usage examples, see [GPIO usage ](../../../usage/gpio/README.md). When reading this page, reference should be made to the BCM2835 ARM peripherals [data sheet](../bcm2835/README.md), section 6.
 
-GPIO pins can be configured as either general-purpose input, general-purpose output or as one of up to 6 special alternate settings, the functions of which are pin-dependent.
+GPIO pins can be configured as either general-purpose input, general-purpose output, or as one of up to six special alternate settings, the functions of which are pin-dependent.
 
-There are 3 GPIO banks on BCM2835.
+There are three GPIO banks on BCM2835.
 
-Each of the 3 banks has its own VDD input pin. On Raspberry Pi, all GPIO banks are supplied from 3.3V. **Connection of a GPIO to a voltage higher than 3.3V will likely destroy the GPIO block within the SoC.**
+Each of the three banks has its own VDD input pin. On Raspberry Pi, all GPIO banks are supplied from 3.3V. **Connection of a GPIO to a voltage higher than 3.3V will likely destroy the GPIO block within the SoC.**
 
 A selection of pins from Bank 0 is available on the P1 header on Raspberry Pi.
 
-## GPIO Pads
+## GPIO pads
 
-The GPIO connections on the BCM2835 package are sometimes referred to in the peripherals datasheet as "pads" - a semiconductor design term meaning "chip connection to outside world".
+The GPIO connections on the BCM2835 package are sometimes referred to in the peripherals datas heet as "pads" — a semiconductor design term meaning 'chip connection to outside world'.
```

Comment:
I guess `datas heet` here was supposed to be `data sheet` ?

Reply:
...with a proposed fix (not by me) in #885 :wink:

Reply:
Merged /cc @JaninaPi